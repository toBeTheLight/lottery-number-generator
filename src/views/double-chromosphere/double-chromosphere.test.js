import React from 'react';
import ReactDom from 'react-dom';
import DoubleChromosphere from './double-chromosphere';
import { mount } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';
import { HashRouter as Router} from 'react-router-dom';

describe('', () => {
  it('rdouble-chromosphere', () => {
    const wrapper = mount(
      <Router>
        <DoubleChromosphere />
      </Router>);
    expect(renderToJson(wrapper)).toMatchSnapshot();
  })
})