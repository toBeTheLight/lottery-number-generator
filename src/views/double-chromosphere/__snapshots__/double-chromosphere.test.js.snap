// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rdouble-chromosphere 1`] = `null`;
